src/views/List.css

```diff
@@ -0,0 +1,97 @@
+/* Empty list view CSS */
+.empty-list-view .card {
+	width: 345px;
+	text-align: center;
+	border: 0;
+	background-color: transparent;
+	margin: auto;
+}
+
+.empty-list-view .card-img-overlay {
+	padding-top: 25%;
+	text-align: center;
+}
+
+.empty-list-view .oven-handle {
+	border-color: #495867;
+	border-width: 1.25px;
+	border-style: solid;
+	border-radius: 5px;
+	padding: 10px;
+	margin-bottom: 5%;
+}
+
+.empty-list-view .card-overlay-background {
+	background: #ffe2cd;
+	border-color: #495867;
+	border-width: 1.25px;
+	border-style: solid;
+	border-radius: 5px;
+	padding: 5%;
+}
+
+.empty-list-view .card-img-overlay img {
+	width: 170px;
+	margin-bottom: 5%;
+}
+
+.empty-list-view .card-title {
+	margin-bottom: 5%;
+}
+
+.empty-list-view .card-subtitle {
+	margin-bottom: 5%;
+}
+
+.empty-list-view button {
+	margin-bottom: 5%;
+}
+
+/* Filled shopping list view CSS */
+.filled-list-view .search-bar {
+	margin: auto;
+	width: 275px;
+	padding-top: 15px;
+	padding-bottom: 15px;
+}
```

Comment:
Just a heads up that the search bar is very slightly off center.

Reply:
Yes! Agreed I saw this too but I believe this will be remedied with the global centering CSS of index in issue #41 - as it was fixed when working with Randall on a rebase that merged both of the respective PRs.